android/src/main/java/com/rudderstack/react/android/RNRudderSdkModule.java

```diff
@@ -88,6 +89,21 @@ public void screen(String event, ReadableMap properties, ReadableMap options) {
                 .build());
     }
 
+    @ReactMethod
```

Comment:
Let's use a method `putDeviceToken` specific to this platform instead of keeping separate methods and we should control this platform-specific behavior from TS code.

Ref: https://reactnative.dev/docs/platform-specific-code

Reply:
done